release notes/v0.54.0.md

```diff
@@ -0,0 +1,304 @@
+
+### Support ArrayBufferViews in send for `k6/experimental/websockets` [#3944](https://github.com/grafana/k6/pull/3844)
+
+As part of making `k6/experimental/websockets` complaint with the [WebSocket API](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket) it now support Uint8Array and other ArrayBufferViews directly as arguments to `send`, instead of having to specifically provide their `buffer`.
+
+This should make it more compliant with libraries that use WebSocket API.
+
+Thanks to @pixeldrew for [reporting](https://github.com/grafana/xk6-websockets/issues/75) this :bow:
+
+
+### Rework of how usage is being collected internally and additional counters [`#3917`](https://github.com/grafana/k6/pull/3917) and [`#3951`](https://github.com/grafana/k6/pull/3951)
+
+As part of working on k6 over the years we have many times wondered if given features is used or if this strange corner case behavior is being used.
+
+We usually made guesses or tried to extrapolate from experience on issues we see or maybe even cloud customers. This unfortunately isn't always easy or possible at all and definitely is very skewed. As such we usually also added warning messages to thing we intend on breaking to let people know and ask them to report if they see problems. This usually see very little activity especially before we make the changes.
+
+This also only works for things we want to remove, but doesn't help us know if people use new functionality at all or if there are patterns that we do not expect.
+
+While k6 has been collecting usage for a while it is usually very surface level stuff,  such as how many VUs were run, k6 version, what *internal* modules were loaded and such.
+
+And the system for this was very rigid requiring usually a lot of work to add simple stuff, like if someone used the `require` function.
+
+This has been reworked to make this a lot easier to do and a few new usage reports have been added:
+- when cloud is used it will also tell us which test run it is. We already have most if not actually more of the information reported through metrics and it being ran in the cloud. But this will also help us filter cloud from not cloud runs easier and also potentially tell cloud users that they are using experimental modules that are to be removed soon.
+- number of files parsed as well as number of ts files parsed. This will be useful both in us figuring out if people use small projects or bigger ones as well as if ts file support is being used.
+- usage of `require`. Now that we have ESM native support, using `require` and CommonJS adds complexity. It is interesting to us whether removing this in the future - likely years, given it support in other runtimes, is even feasable.
+- usage of `global`. This for help us decide if we can [drop compatility-mode](https://github.com/grafana/k6/issues/3864) differences or even the whole concept of them.
```

Comment:
```suggestion
While k6 has been collecting usage for a while, they're surface-level things, such as how many VUs were run, the k6 version, or which *internal* modules were loaded. The system for this was also very rigid, requiring a lot of work to add simple things, such as if someone used the `require` function.

This process has been reworked to make things easier, and a few new usage reports have been added:

- When Grafana Cloud is used, it will also tell us which test run it is. We already have most of the information reported through metrics and from the test being executed in the cloud. But this will help us filter cloud test runs from local runs and potentially warn cloud users if they're using experimental modules that will be removed.
- The number of files parsed, as well as the number of .ts files parsed. This will help us understand if people use small or big projects and if TypeScript support is being used.
- Usage of `require`. Now that we have ESM native support, using `require` and CommonJS adds complexity. It's interesting to us whether removing this in the future - likely years, given its support in other runtimes, is feasible.
- Usage of `global`. This will help us decide if we can [drop compatibility-mode](https://github.com/grafana/k6/issues/3864) differences or even the whole concept.
```